tools/install_demo_configuration.sh

```diff
@@ -378,6 +378,280 @@ echo "opendistro_security.enable_snapshot_restore_privilege: true" | $SUDO_CMD t
 echo "opendistro_security.check_snapshot_restore_write_privileges: true" | $SUDO_CMD tee -a "$ES_CONF_FILE" > /dev/null
 echo 'opendistro_security.restapi.roles_enabled: ["all_access", "security_rest_api_access"]' | $SUDO_CMD tee -a "$ES_CONF_FILE" > /dev/null
 
+#configure default LDAP
+if [ "$EnableLDAP" == true ]; then
+    echo "Configure LDAP anthentication"
+    OP_DISTRO_CONFIG_FILE=/usr/share/elasticsearch/plugins/opendistro_security/securityconfig/config.yml
```

Comment:
Please use [plugin dir variable](https://github.com/opendistro-for-elasticsearch/security/blob/717c5c285cf94cb16573da13cc4cc75f5bc8d4a6/tools/install_demo_configuration.sh#L109) instead of putting a path here. This path can change depending on the type of installation.